### ISSUE TITLE
Quality OOB with long reads

### ISSUE THREAD

**Issue by mwiewior** [biodatageeks/sequila#112]
```bash
21/11/14 18:30:48 INFO NewHadoopRDD: Input split: file:/Users/mwiewior/research/data/long_reads/rel5-guppy-0.3.0-chunk10k.wgs.proper.chr1.bam:2199850975587-4398046576639
21/11/14 18:30:48 INFO AlignmentsRDD: Finishing reading partition with read e9df66ae-a308-4c15-ae59-62e6c5fb4027, chr1:818666
21/11/14 18:30:48 ERROR Executor: Exception in task 0.0 in stage 4.0 (TID 646)
java.lang.ArrayIndexOutOfBoundsException: Index 3562 out of bounds for length 264
        at org.biodatageeks.sequila.pileup.model.AggregateRDD.fillBaseQualities(ContigAggregateRDD.scala:184)
        at org.biodatageeks.sequila.pileup.model.AggregateRDD.prepareOutputQualMap(ContigAggregateRDD.scala:199)
        at org.biodatageeks.sequila.pileup.model.AggregateRDD.addBaseRecord(ContigAggregateRDD.scala:162)
        at org.biodatageeks.sequila.pileup.model.AggregateRDD.$anonfun$toPileup$3(ContigAggregateRDD.scala:63)
        at org.biodatageeks.sequila.pileup.model.AggregateRDD.$anonfun$toPileup$2(ContigAggregateRDD.scala:55)
        at scala.collection.Iterator$$anon$10.next(Iterator.scala:461)
        at scala.collection.Iterator$$anon$11.nextCur(Iterator.scala:486)
        at scala.collection.Iterator$$anon$11.hasNext(Iterator.scala:492)
        at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.agg_doAggregateWithoutKey_0$(Unknown Source)
        at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
        at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
        at org.apache.spark.sql.execution.WholeStageCodegenExec$$anon$1.hasNext(WholeStageCodegenExec.scala:755)
        at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
        at org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:132)
        at org.apache.spark.shuffle.ShuffleWriteProcessor.write(ShuffleWriteProcessor.scala:59)
        at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:99)
        at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:52)
        at org.apache.spark.scheduler.Task.run(Task.scala:131)
        at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$3(Executor.scala:497)
        at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:1439)
        at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:500)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:829)
21/11/14 18:30:48 INFO TaskSetManager: Starting task 1.0 in stage 4.0 (TID


```

**Comment by mwiewior** [comment]
First partition - easy to reproduce
```
chr1:818666
```

**Comment by mwiewior** [comment]
Fixed by #122
